### DISCUSSION TITLE
Resizing PDF

### DISCUSSION THREAD

**Discussion by EthanC-8** [pymupdf/PyMuPDF#2783] [Looking for help] [answered]
Hey there, basically I got a PDF which i want to resize by 1/4.. issue is I want to resize but keep the page on orignal 
I am guessing I will have to resize and then put it on a new pdf's new page or something? I am super confused

So far I got this..

```
    src = fitz.open(file_name)
    doc = fitz.open()  # new output PDF
    for page in src:
        w, h = page.rect.br  # extract input page widht and height
        newpage = doc.new_page(width=w/4, height=h/4)
        newpage.show_pdf_page(newpage.rect, src, page.number)
    doc.save(file_name)
```
this resizes the whole PDF but i want it to be on a new page.. something like that image i attached as an example 

![Untitled](https://github.com/pymupdf/PyMuPDF/assets/33990363/73cb6841-e9c2-4e99-8199-431bc10776cf)

**Comment by JorjMcKie** [comment]
When you say "resize":
Do you mean shrink the page dimensions but show the full old page content in that miniature page?
Or: Do you want to appear only the top-left quart of the old pages' contents?

**Reply by EthanC-8** [reply]
for example i want the orignal one to become 25% of it's size and and then put it on top left on a new paper so it has white space..
what i want to achieve is bascialyl so that when someone prints it on paper it is not taking the whole page but only 1/4

**Reply by JorjMcKie** [reply, marked as answer]
I hope this is what you mean:
```python
src = fitz.open("input.pdf")
out = fitz.open()  # the output document
for src_page in src:  # iterate over input pages
    prect = src_page.rect  # page rectangle
    # make empty output page
    out_page = out.new_page(width=prect.width, height=prect.height)  # same size as input page
    # compute rectangle for miniature input page
    prect /= 4  # has width / height 1 quarter of original
    out_page.show_pdf_page(prect, src, src_page.number)
out.ez_save("output.pdf")  # save output file
```

**Comment by EthanC-8** [comment]
It worked <3
